lrpar/src/lib/ctbuilder.rs

```diff
@@ -360,15 +360,19 @@ where
         let inc = read_to_string(grmp).unwrap();
 
         let grm = YaccGrammar::<StorageT>::new_with_storaget(yk, &inc).map_err(|e| {
-            let mut line_cache = NewlineCache::new();
-            line_cache.feed(&inc);
-            if let Some((line, column)) =
-                line_cache.byte_to_line_num_and_col_num(&inc, e.span.start())
-            {
-                format!("{} at line {line} column {column}", e)
-            } else {
-                format!("{}", e)
+            let mut s = String::new();
+            for e in e {
```

Comment:
I'm not sure I like `for e in e` ;)

A possibly bigger problem is that I *think* it leaves one too many `\n` at the end of the string?

Could we simplify to something like:

```rust
errs.map(|e| if let Some(...) = ... { format!(...) } else { format!(...) }).join("\n")
```

[The type checker might need a `collect::<Vec<_>>()` (or similar in) the middle.]

Reply:
fixed in edc3d0fbeb9f3711ca3380cef0ea24f4bd4ba660